engine/release-notes/23.0.md

```diff
@@ -41,6 +41,28 @@ Changing the version format is a stepping-stone towards Go module compatibility,
 but the repository doesn't yet use Go modules, and still requires using a "+incompatible" version.
 Work continues towards Go module compatibility in a future release.
 
+## 23.0.5
+
+{% include release-date.html date="2023-04-26" %}
+
+For a full list of pull requests and changes in this release, refer to the relevant GitHub milestones:
+
+- [docker/cli, 23.0.5 milestone](https://github.com/docker/cli/milestone/79?closed=1)
+- [moby/moby, 23.0.5 milestone](https://github.com/moby/moby/milestone/118?closed=1)
+
+### Bug fixes and enhancements
+
+- Add the --all / -a option when pruning volumes. [docker/cli#4229](https://github.com/docker/cli/pull/4229)
+- Add --format=json for `docker info`. [docker/cli#4320](https://github.com/docker/cli/pull/4230)
+- Fix log loss with the AWSLogs log driver. [moby/moby#45350](https://github.com/moby/moby/pull/45350)
+- Fix a regression introduced in v23.0.4 where dockerd would refuse to start if the fixed-cidr config parameter is provided but not bip. [moby/moby#45403](https://github.com/moby/moby/pull/45403)
+- Fix a panic in libnetwork during daemon start [moby/moby#45376](https://github.com/moby/moby/pull/45376)
+- Fix "tag" event not being sent when an image is built with `buildx`. [moby/moby#45410](https://github.com/moby/moby/pull/45410)
```

Comment:
Moving the full-stop to the end here as well;

```suggestion
- Fix log loss with the AWSLogs log driver [moby/moby#45350](https://github.com/moby/moby/pull/45350).
- Fix a regression introduced in v23.0.4 where dockerd would refuse to start if the fixed-cidr config parameter is provided but not bip [moby/moby#45403](https://github.com/moby/moby/pull/45403).
- Fix a panic in libnetwork during daemon start [moby/moby#45376](https://github.com/moby/moby/pull/45376).
- Fix "tag" event not being sent when an image is built with `buildx` [moby/moby#45410](https://github.com/moby/moby/pull/45410).
```

Reply:
Oh, actually, looks like we've not been consistent with the full-stop; @dvdksn any preference?

- Foo bar hello world. [moby/moby#1234](link)

or


- Foo bar hello world [moby/moby#1234](link).

Reply:
Yeah, I looked and the 23.0.4 release notes and did the same :)

Reply:
I prefer the first one, personally

Reply:
👍 marked this as resolved then 👍